pkg/apis/core/validation/validation.go

```diff
@@ -5579,6 +5579,11 @@ func ValidateNamespace(namespace *core.Namespace) field.ErrorList {
 	for i := range namespace.Spec.Finalizers {
 		allErrs = append(allErrs, validateFinalizerName(string(namespace.Spec.Finalizers[i]), field.NewPath("spec", "finalizers"))...)
 	}
+	if utilfeature.DefaultFeatureGate.Enabled(features.NamespaceDefaultLabelName) {
```

Comment:
note to self (or @abhiraut whoever gets to it first :)) we need to remove feature gates, we're going striaght to GA :)   per original PR that is mostly done now